Private/Invoke-Remediation.ps1

```diff
@@ -0,0 +1,273 @@
+function Invoke-Remediation {
+    <#
+    .SYNOPSIS
+    Runs any remediation scripts available.
+
+    .DESCRIPTION
+    This function offers to run any remediation code associated with identified issues.
+
+    .PARAMETER AuditingIssues
+    A PS Object containing all necessary information about auditing issues.
+
+    .PARAMETER ESC1
+    A PS Object containing all necessary information about ESC1 issues.
+
+    .PARAMETER ESC2
+    A PS Object containing all necessary information about ESC2 issues.
+
+    .PARAMETER ESC3
+    A PS Object containing all necessary information about ESC3 issues.
+
+    .PARAMETER ESC4
+    A PS Object containing all necessary information about ESC4 issues.
+
+    .PARAMETER ESC5
+    A PS Object containing all necessary information about ESC5 issues.
+
+    .PARAMETER ESC6
+    A PS Object containing all necessary information about ESC6 issues.
+
+    .INPUTS
+    PS Objects
+
+    .OUTPUTS
+    Console output
+    #>
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $false)]
```

Comment:
I believe the best, simplest practice would be to exclude `Mandatory = $false` as that is already the default value.

Reply:
Done. Removed all instances of it.